src/tsserver/nodeServer.ts

```diff
@@ -717,7 +871,7 @@ function startNodeSession(options: StartSessionOptions, logger: Logger, cancella
     }
 
     const ioSession = useNodeIpc ? new IpcIOSession() : new IOSession();
-    process.on("uncaughtException", err => {
+    process.on("uncaughtException", (err) => {

```

Comment:
There's nothing being fixed here, you're just changing from one style to another